src/nwb/hdmf/base/Data.hpp

```diff
@@ -2,7 +2,7 @@
 
 #include <memory>
 
-#include "BaseIO.hpp"
+#include "../../../BaseIO.hpp"
```

Comment:
To be honest, I find full paths within the library more intuitive, in particular when we have to cross multiple levels in the folder hierarchy.